sdk/inc/azure/storage/az_storage_blobs.h

```diff
@@ -128,7 +125,7 @@ AZ_NODISCARD AZ_INLINE az_storage_blobs_blob_upload_options
 az_storage_blobs_blob_upload_options_default()
 {
   return (az_storage_blobs_blob_upload_options){ .context = &az_context_application,
-                                                 ._internal = { .unused = AZ_SPAN_NULL } };
+                                                 ._internal = { .unused = true } };
```

Comment:
`unused` should be set to false as that is what the "default" value would be if we did:
```C
az_storage_blobs_blob_upload_options options = { 0 };
```

Reply:
```suggestion
                                                 ._internal = { .unused = false } };
```